admin/listPurchases/listView.html

```diff
@@ -135,12 +133,15 @@
     }
 }
 
-function ShiftedStartIndexToRender(amountOfSubpages){
-    if(currentPageIndex + Math.floor(amountOfShownSubpages/2) >= amountOfSubpages){
-        return amountOfSubpages - amountOfShownSubpages;
-    }
-    if(currentPageIndex - Math.floor(amountOfShownSubpages/2) > 0){
-        return currentPageIndex - Math.floor(amountOfShownSubpages/2);
+function calculateStartIndexToRender(amountOfSubpages, extraNavNeeded){
+    if(extraNavNeeded){
+        const flooredHalfAmountOfShowSubages = Math.floor(amountOfShownSubpages/2);
```

Comment:
This variable name just says what the right side says, so it adds no explanation.

What would be a nice shorthand you would use when explaining this code, so you don't have to say "floored half amount of shown subpages" every time you refer to this number? Use that as the variable name instead.